src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIResponse.java

```diff
@@ -48,6 +50,7 @@
   private int httpStatus;
   private Map<String, String> headers;
   private String responseBody;
+  private boolean isRetriable;
```

Comment:
rename to retryable

Reply:
Renamed to `retryable` here and at other places. a752f6b05f68ae64ce9ed63c94fc7b1cc1865b12